### ISSUE TITLE
Resolution is not saved

### ISSUE THREAD

**Issue by AlanRibas** [libretro/Mesen#6]
I was configuring the core of the MESEN emulator in the retroarch.
I made several changes and saved them in the end. But the resolution refuses to change.
More than once I changed the resolution that is currently 1920x1080 to 1280x720.
The moment the game is open the resolution is changed. But just leaving retroarch to lose that configuration.

**Comment by Sanaki** [comment]
This is a RetroArch issue, not a Mesen issue. The core itself has no resolution settings and thus no effect on anything the frontend may do in that regard. There's also a decent chance it's simply a problem with your overrides, as opposed to an actual problem with RetroArch.

**Comment by AlanRibas** [comment]
> This is a RetroArch issue, not a Mesen issue. The core itself has no resolution settings and thus no effect on anything the frontend may do in that regard. There's also a decent chance it's simply a problem with your overrides, as opposed to an actual problem with RetroArch.

if the Mesen kernel does not have the option to set the resolution perhaps then a future update of Retroarch should bring the resolution option disabled for tak kernel.  and Preferably stating the reason why other users do not think it is a problem.

as for the part of being a configuration problem.  I do not know that.  As I said above, the settings made are saved with the exception of the resolution part.  if it was a problem with my computer I think there would be a problem saving the settings as a whole and not just a single part

**Comment by Sanaki** [comment]
You're misunderstanding. This is the wrong repo to open the issue in, and thus this issue should be closed. Resolution is not relevant to this core at all. NES only outputs one resolution. While the HD packs supported by the core can use higher resolution assets, that doesn't necessitate resolution handling by the core itself. If it's a RetroArch issue, it needs to be opened in the RetroArch repo. As for whether it's an issue at all, I'm having difficulty even understanding what you're having issues with. It -sounds- like you're trying to change your fullscreen resolution when running this core for some reason. If so, you just change those options in retroarch and save a core override. There are multiple resolution options, you may have changed the wrong ones. It's also possible those options don't work from an override, in which case you can open an issue on the RetroArch issue tracker, not this one, asking for that to be addressed.

**Comment by AlanRibas** [comment]
thank you very much for the explanations.  I did not understand.  I'll open a post there then.  just to clarify it.  I started to change the resolution of the 8 and 16 bit systems that I have on my computer.  I recently had a problem with Master system games running with the Gen X plus core ... I think that's the name ... what happened that left all these systems configured to run on full HD.  however, this caused a problem with the game image.  some effects did not behave the way they should.  apart from the fact that the transition of the background images does not become fluid.  I just had to change the resolution and all the problems that I was going through were solved.  that's the reason.


Thank you very much for your help.!!!
